ChaosMod/Components/EffectDispatcher.cpp

```diff
@@ -566,6 +569,16 @@ void EffectDispatcher::ClearMostRecentEffect()
 	}
 }
 
+std::vector<RegisteredEffect*> EffectDispatcher::GetRecentEffects(int distance)
```

Comment:
Nitpick but this could be marked as a const function as it doesn't modify any fields (`const` at the end of the line, also in the header)